### PR TITLE
KOGITO-1822 - Create serverless greeting example for kogito and sprin…

### PR DESCRIPTION
**Pull request by tsurdilo**
…g boot

### CODE REVIEW

**Comment by tsurdilo** [comment]
DEPENDS ON https://github.com/kiegroup/kogito-runtimes/pull/437

**Comment by ricardozanini** [comment]
Just to clarify, Jenkins error is caused by this PR that we depend on: https://github.com/kiegroup/kogito-runtimes/pull/437

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
Tests OK

**Review by cristianonicolai — APPROVED** [review]

File: serverless-workflow-greeting-springboot/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+```sh
+mvn clean package
+```
+
+To run the generated native executable, generated in `target/`, execute
+
+```sh
+java -jar target/serverless-workflow-greeting-springboot.jar
+```
+
```
Comment (jstastny-cz) [inline serverless-workflow-greeting-springboot/README.md:48]: If the example is based on CURL requests and we're also providing swagger.json I would put here the instructions for what swagger is and where it's located, see other springboot examples for the wording.

**Review by jstastny-cz — CHANGES_REQUESTED** [review]

File: serverless-workflow-greeting-springboot/README.md (hunk 1 above)
Comment (tsurdilo) [reply serverless-workflow-greeting-springboot/README.md:48]: just curl. thanks.

Comment (jstastny-cz) [reply serverless-workflow-greeting-springboot/README.md:48]: In that case please put here the instructions for Swagger (as is in other Spring Boot examples - mind that there's difference in those instructions between quarkus and boot examples). Usually it is after Packaging, before the first example.

**Review by evacchi — APPROVED** [review]

File: serverless-workflow-greeting-springboot/README.md (hunk 1 above)
Comment (tsurdilo) [reply serverless-workflow-greeting-springboot/README.md:48]: added

**Comment by tsurdilo** [comment]
@jstastny-cz added swagger info to readme. thanks!

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**